garak/probes/leakreplay.py

```diff
@@ -11,6 +11,8 @@
 from garak import _config
 from garak.probes.base import Probe
 
+resources = _config.transient.basedir / "resources"
```

Comment:
precision of `resources` as a var name could be increased
```suggestion
resources_path = _config.transient.basedir / "resources"
```